src/common/const.ts

```diff
@@ -139,6 +140,19 @@ export const DOMAINS_HIDE_MORE_INFO = [
   "select",
 ];
 
+/** Domains that render a special element instead of a text value when rendered in a row.
+ *  Those rows should then not show a cursor pointer when hovered (which would normally
+ *  be the default) unless the element itself enforces it (e.g. a button). Also those elements
+ *  should not act as a click target to open the more info dialog (the row name and state icon
+ *  still do of course) as the click might instead e.g. activate the input field that this row shows.
+ */
+export const DOMAINS_NO_POINTER_CLICK_IN_ROW = [
```

Comment:
Not in love with that constant name, but the best I could come up with 😬. Ideas welcome!